src/main/java/spark/route/Routes.java

```diff
@@ -16,11 +16,7 @@
  */
 package spark.route;
 
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
🤔  wildcard?

Reply:
As per the author request, IntelliJ is used.